storage-nvmf/src/main/java/org/apache/crail/storage/nvmf/NvmfStorageServer.java

```diff
@@ -86,15 +100,15 @@ public void run() {
 	@Override
 	public StorageResource allocateResource() throws Exception {
 		StorageResource resource = null;
-		
+
 		if (alignedSize > 0){
 			LOG.info("new block, length " + NvmfStorageConstants.ALLOCATION_SIZE);
-			LOG.debug("block stag 0, offset " + offset + ", length " + NvmfStorageConstants.ALLOCATION_SIZE);
+			LOG.debug("block stag 0, address " + address + ", length " + NvmfStorageConstants.ALLOCATION_SIZE);
 			alignedSize -= NvmfStorageConstants.ALLOCATION_SIZE;
-			resource = StorageResource.createResource(offset, (int)NvmfStorageConstants.ALLOCATION_SIZE, 0);
-			offset += NvmfStorageConstants.ALLOCATION_SIZE;
+			resource = StorageResource.createResource(address, (int)NvmfStorageConstants.ALLOCATION_SIZE, 0);
```

Comment:
#sad. Why not make ALLOCATION_SIZE an integer to start with. It is a long which is bad because a user might be tempted to give it a > 2GB values considering it is a long.

Reply:
Same goes for other NvmfStorageConstants constants as well.

Reply:
Good point. Not sure why I made them long in the first place.